about/troubleshooting.rst

```diff
@@ -38,6 +38,18 @@ There are several ways to improve performance and battery life:
   This way, it will be hidden in the editor but will still be visible in the
   running project.
 
+The editor stutters and flickers on my variable refresh rate monitor (G-Sync/FreeSync).
+---------------------------------------------------------------------------------------
+
+This is a `known issue <https://github.com/godotengine/godot/issues/38219>`__.
+There are two workarounds for this:
+
+- Enable **Interface > Editor > Update Continuously** in the Editor Settings. Keep in mind
+  this will increase power usage and heat/noise emissions, since the editor will
+  now be rendering constantly, even if nothing has changed on screen. To alleviate this,
+  you can increase **Low Processor Mode Sleep Usec** in the Editor Settings.
```

Comment:
Could we give users a recommended value of **Low Processor Mode Sleep Usec** it's unclear how much should you increase it to. Mine is 6900 as a default are we talking increase it to 7000 or 70 000? Would be good idea to give them some guidance on what sort of degree of scale are we talking about. Apart of that it looks solid.